### PR TITLE
Add file version remove

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| License | MIT

#### What's in this PR?

Add file version remove.

#### Why?

To remove a specific file version.

#### To Do

- [x] Implement Controller
- [x] Implement JS
   - [x] Implement Confirmation Overlay?
- [x] Changelog
- [x] Tests

### CODE REVIEW

**Comment by alexander-schranz** [comment]
@chirimoya how do we implement the confirmation of the delete button? I think we will not create a overlay in overlay? Currently I only see the native confirmation but we didn't use that elsewhere and think the designers will be angry then.

delete-version

/cc @berndhep

File: src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js [hunk 1]
```diff
@@ -648,6 +689,8 @@ define([
             this.sandbox.off('husky.tabs.overlaymedia-edit.item.select');
             this.sandbox.off('husky.dropzone.file-version.files-added');
             this.sandbox.off('husky.dropzone.preview-image.files-added');
+            this.sandbox.dom.off(this.$el, 'click.version-remove');
+            this.sandbox.dom.off(this.$el, 'click.clipboard-copy');
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js]: :+1:

File: src/Sulu/Bundle/MediaBundle/Resources/translations/sulu/backend.en.xlf [hunk 2]
```diff
@@ -414,6 +414,10 @@
                 <source>sulu.media.audience-target-groups</source>
                 <target>Audience target groups</target>
             </trans-unit>
+            <trans-unit id="1c4135ca12b79a2w4caj83643e831812a" resname="sulu-media.restore-version">
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/MediaBundle/Resources/translations/sulu/backend.en.xlf]: really needed?

**Comment by alexander-schranz** [comment]
@wachterjohannes feel free to make a  suggestion.

/cc @chirimoya

**Pull request merged**